app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -108,10 +108,13 @@ class LoginFragment : Fragment() {
 
     private fun redirectToMain(bundle: Bundle?) {
         val intent = Intent(activity, MainActivity::class.java)
-        if (((bundle != null) && !id.equals(-1)) && (ticketIdAndQty != null)) {
-            intent.putExtra(LAUNCH_ATTENDEE, true)
+        if (bundle != null) {
+            if ((!id.equals(-1)) && (ticketIdAndQty != null)) {
```

Comment:
not required, these 2 if statements can be combined just like it was before

Reply:
Remove unneeded paranthesis

Reply:
> Remove unneeded parenthesis

Done :+1:

Reply:
> Remove unneeded parenthesis

Done :+1: